website/docs/r/policy_rule_idp_discovery.html.markdown

```diff
@@ -76,6 +110,17 @@ The following arguments are supported:
 
 - `idp_type` - (Optional) Type of Idp. One of: `"SAML2"`, `"IWA"`, `"AgentlessDSSO"`, `"X509"`, `"FACEBOOK"`, `"GOOGLE"`, `"LINKEDIN"`, `"MICROSOFT"`, `"OIDC"`
 
+- `idp` - (Optional) List of multiple Idp associated with this resource. If set this will overwrite the `idp_id` and `idp_type` attributes.
+
+  - `id` - (Required) The identifier for the Idp this rule should route to.
+  - `type` - (Optional) Type of Idp. One of: `"SAML2"`, `"IWA"`, `"AgentlessDSSO"`, `"X509"`, `"FACEBOOK"`, `"GOOGLE"`, `"LINKEDIN"`, `"MICROSOFT"`, `"OIDC"`
+
+````hcl
```

Comment:
Code blocks need to be three ticks
```hcl
# words
```
You can double check how documentation renders here https://registry.terraform.io/tools/doc-preview